packages/grafana-ui/src/components/ContextMenu/ContextMenu.tsx

```diff
@@ -11,14 +11,16 @@ export interface ContextMenuProps {
   y: number;
   /** Callback for closing the menu */
   onClose?: () => void;
+  /** Enable focus on first Menu element on openning the menu */
+  focusOnOpen?: boolean;
```

Comment:
The changes for ContextMenu should be in their own PR -- so the simple behavior change is clearly noted in the commit history.